### PR TITLE
Grid component

### LINKED ISSUE #3: Grid
## Figma
https://www.figma.com/design/EjFt1XdacOogDZoQ7RetrI/Frequency-Style-Guidelines?node-id=823-747&node-type=frame&t=TSl6Zk6gudf4G3VM-0

### PR DESCRIPTION
**Pull request by shannonwells**
# Problem
Add Grid component with breakpoints, columns, gutter & margins according to design specification.

Closes #3 

Copies method for sizing from [Frequency.xyz](https://frequency-chain/frequency-xyz)

## Steps to Verify:
`npm run storybook`
Go to Design System --> Grid and check out different sizes.
Compare with Figma design linked in Issue.

## Screenshots (optional):
Grid

### CODE REVIEW

File: src/lib/consts.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const MAX_PAGE_WIDTH: number = 1700;
```
Comment (aramikm) [inline src/lib/consts.ts:1]: nit: Do we need units in here?

**Review by aramikm — APPROVED** [review]
Tested on **Chrome** and **Firefox** and **Safari** and it looked good!

File: src/lib/consts.ts (hunk 1 above)
Comment (claireolmstead) [reply src/lib/consts.ts:1]: I think in front end it's pretty normal to assume it's pixels

File: src/lib/features/Grid.stories.svelte [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    'consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source.',
+    'Quisque vitae ipsum enim.',
+    'Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. ',
+    'Aliquam lobortis nisl mi, et ultrices mi gravida ac.',
+  ];
+</script>
+<Meta title="Design System/Grid" />
+Resize window or activate responsive sizing in the development panel to see grid changes.
+<Story name="Grid">
+  <Grid class="xs:text-xs md:text-sm">
```
Comment (claireolmstead) [inline src/lib/features/Grid.stories.svelte:23]:
```suggestion
  <Grid class="xs:xs md:sm">
```

File: src/lib/features/Grid.svelte [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<style>
+</style>
+<div class="m-4 md:m-8 lg:m-32 {$$restProps.class}">
+<div class="grid grid-cols-4 gap-4 sm:grid-cols-4 md:gap-8 md:grid-cols-6 lg:grid-cols-12 lg:gap-8">
```
Comment (claireolmstead) [inline src/lib/features/Grid.svelte:4]: opt for using the "f..." spacing units from the styleguide

**Review by claireolmstead — APPROVED** [review]
Small changes, but looks good!!

**Pull request merged**